templates/default/disable_ipv6_loopback.sh.erb

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Disable the ipv6 loopback address to make chef server and EMPD happy
+if ! grep net.ipv6.conf.lo.disable_ipv6 /etc/sysctl.conf; then
+  echo "net.ipv6.conf.lo.disable_ipv6=0" | sudo tee -a /etc/sysctl.conf
```

Comment:
On both base images we enable ipv6, but on CentOS you don't actually have to set the loopback flag to have an IPv6 address. This just adds the param to please the preflight check.